rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

```diff
@@ -84,6 +84,33 @@ Change your own password regularly. You will be logged out in the process and ha
 . Click *OK*.
 . Log in with your new password.
 
+[[iforgotmypassword]]
+== Reset Your Password
+
+If you forget your password or if your one-time password expires, click the link on the RPA Manager login page to reset it. You need to know your user name to reset your password.
+
+. Navigate to the RPA Manager login screen.
+. Click *Forgot your credentials?*.
+. In the *Forgot your credentials* form, enter your username.
+. Click *Continue*.
+
+A confirmation request for creating a new password is emailed to you.
+
+. Open the email in your email application.
+. Click *Request New Credentials*.
+
+A link to create a new password is emailed to you. The one-time password expires after 24 hours.
+If you use Two-Factor Authentication, append the time-based one-time password to your one-time password.
```

Comment:
I don't think you need to use capital letters for ˝two-factor authentication"

Reply:
changed